src/runtime_src/core/common/info_memory.cpp

```diff
@@ -411,4 +413,190 @@ xclbin_info(const xrt_core::device * device)
   return pt;
 }
 
+enum class cu_type 
+{
+  pl,
+  ps
+};
+
+static std::string 
+enum_to_str(cu_type type) 
+{
+  switch (type) {
+    case cu_type::pl:
+      return "PL";
+    case cu_type::ps:
+      return "PS";
+  }
+  return "UNKNOWN";
+}
+
+ptree_type
+get_cu_status(uint32_t cu_status)
+{
+  ptree_type pt;
+  std::vector<std::string> bit_set;
+  if (cu_status & 0x1)
+    bit_set.push_back("START");
+  if (cu_status & 0x2)
+    bit_set.push_back("DONE");
+  if (cu_status & 0x4)
+    bit_set.push_back("IDLE");
+  if (cu_status & 0x8)
+    bit_set.push_back("READY");
+  if (cu_status & 0x10)
+    bit_set.push_back("RESTART");
+
+  pt.put("bit_mask",	boost::str(boost::format("0x%x") % cu_status));
```

Comment:
Format: extra spaces (e.g., a tab use used)
```suggestion
  pt.put("bit_mask", boost::str(boost::format("0x%x") % cu_status));
```